03-make/make_homework/Makefile

```diff
@@ -0,0 +1,14 @@
+TXT:=cat #computes variable value once and just stores it
+TARGET?=sup.txt #if the variable was not defined
+
+all : stol.o
+	cp $< $(TARGET)
```

Comment:
What is the idea of this command?

Reply:
The comment was added in the code. This just copies stol.o to the $TARGET.
I agree that this extra operation and I can do it directly in the stol.o rule. But I'm learning and wanted to show you I know this command :)